_run/kube/README.md

```diff
@@ -73,97 +66,85 @@ environments.  Two options are offered below - the first (random port) is less e
 and can have multiple instances run concurrently, while the second option arguably
 has a better payoff.
 
-**note**: this step waits for kubernetes metrics to be available, which can take some time.
+**note**: this step waits for Kubernetes metrics to be available, which can take some time.
 The counter on the left side of the messages is regularly in the 120 range.  If it goes beyond 250,
 there may be a problem.
 
-#### Map a random local port to port 80 of your workload
-
-This is less error-prone, but makes it difficult to access your app through the browser.
-
-__t1__
-```sh
-make kind-cluster-create
-```
-
-#### Map local port 80 to to port 80 of your workload
 
-If anything else is listening on port 80 (any other web server), this method
-will fail.  If it does succeed, you will be able to browse your app from the browser.
-
-```sh
-KIND_CONFIG=kind-config-80.yaml make kind-cluster-create
-```
+| Option  | __t1 run step: 1__ | Explanation  |
+|---|---|---|
+| Map random local port to port 80 of your workload | `make kind-cluster-create` | This is less error-prone, but makes it difficult to access your app through the browser. |
+| Map localhost port 80 to workload  | `KIND_CONFIG=kind-config-80.yaml make kind-cluster-create` | If anything else is listening on port 80 (any other web server), this method will fail.  If it does succeed, you will be able to browse your app from the browser. |
 
-Build Akash binaries and initialize network
+## Build Akash binaries and initialize network
 
-__t1__
+__t1 run step: 2__
 ```sh
 make init
 ```
 
-### Run local network
+## Run local network
 
 In a separate terminal, the following command will run the `akashd` node:
 
-__t2__
+__t2 run step: 3__
 ```sh
 make node-run
 ```
 
 You can check the status of the network with:
 
-__t1__
+__t1 status__
 ```sh
-make node-status
+    make node-status
```

Comment:
accidental tab here & in many pre-formatted sections below?

Reply:
I see now that this was intentional.

I *think* this is so you can differentiate status commands from non-status commands, I'm just not sure if that is helpful at all.  Have you seen this elsewhere?

Reply:
I intentionally added the tab as a way to separate status commands from commands which execute a change. 

It's just an additional idea to split the two types of commands. The two are also delineated by the title `__t* run step...__`, so it's a bit redundant.

Reply:
I see.  cool idea.  I was open to it but after looking through it I think it just makes things more confusing.

Every `##` block has one "change" command followed by query commands.  Maybe we just prefix each `##` header with a step number or something.

Reply:
How does `### __t1 Step: 2__` for commands which execute changes and must be run look? eg:

### __t1 Step: 2__
```sh
echo "run me"
```